packages/ui/src/components/SelectInputV2/SelectBar.tsx

```diff
@@ -54,7 +58,7 @@ const Placeholder = styled(Text)`
 user-select: none;
 `
 
-export const StyledInputWrapper = styled(Stack)<{
+const StyledInputWrapper = styled(Stack)<{
```

Comment:
There is an overflowing issue, before:
![Screenshot 2025-05-27 at 10 27 36](https://github.com/user-attachments/assets/6dfd4cb8-5190-4db2-b6a8-bafe9efa3c1f)

Now:
![Screenshot 2025-05-27 at 10 27 50](https://github.com/user-attachments/assets/e048e7a5-f438-407e-ae78-eb8efc67bedc)

It's when I resize the screen maybe you need to add an event listener for that. However it shouldn't overflow the select in any cases.